cupy-xfails.txt

```diff
@@ -14,9 +14,14 @@ array_api_tests/test_array_object.py::test_getitem
 # copy=False is not yet implemented
 array_api_tests/test_creation_functions.py::test_asarray_arrays
 
-# finfo test is testing that the result is a float instead of float32 (see
-# also https://github.com/data-apis/array-api/issues/405)
-array_api_tests/test_data_type_functions.py::test_finfo[float32]
+# attributes are np.float32 instead of float
+# (see also https://github.com/data-apis/array-api/issues/405)
+array_api_tests/test_data_type_functions.py::test_finfo[float32-dtype]
```

Comment:
alternatively to this mess, considering the ongoing controversy in https://github.com/data-apis/array-api/issues/405, we could relax `array-api-tests` to check that the output is either a float or a 0-dimensional real floating array.